exporter/azuremonitorexporter/log_to_envelope.go

```diff
@@ -61,7 +61,8 @@ func (packer *logPacker) LogRecordToEnvelope(logRecord plog.LogRecord) *contract
 
 	messageData.Properties[spanIDTag] = logRecord.SpanID().HexString()
 
-	messageData.Properties[categoryNameTag] = logRecord.Name()
+	// TODO Does this require a replacement?
+	// messageData.Properties[categoryNameTag] = logRecord.Name()
```

Comment:
There is no replacement for LogRecord Name at opentelemetry. I don't see any other options but to delete this and keep the categoryNameTag property unset.

I think it is best to just delete this line since we discourage commented code. You can keep the TODO with an explanation if you prefer (although I do not see how the TODO can be resolved so even that may be unnecessary).